### PR TITLE
Remove dependency on mruby-numeric-ext

### PR DESCRIPTION
**Pull request by tkbky**
Part of GH-32 to remove dependency on mruby-*-ext mrbgems.

### CODE REVIEW

**Comment by lopopolo** [comment]
@tkbky it looks like `mruby-numeric-ext` defines some methods in C:

```c
  mrb_define_method(mrb, i, "chr", mrb_int_chr, MRB_ARGS_NONE());
  mrb_define_method(mrb, i, "allbits?", mrb_int_allbits, MRB_ARGS_REQ(1));
  mrb_define_method(mrb, i, "anybits?", mrb_int_anybits, MRB_ARGS_REQ(1));
  mrb_define_method(mrb, i, "nobits?", mrb_int_nobits, MRB_ARGS_REQ(1));
```

Do you want to take a stab at implementing these in Rust? Otherwise, if you can grant me permission to push to this branch we can work on these together.

File: artichoke-backend/src/extn/core/numeric/numeric.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
```
Comment (lopopolo) [inline artichoke-backend/src/extn/core/numeric/numeric.rb]: LGTM

**Comment by tkbky** [comment]
> Do you want to take a stab at implementing these in Rust?

I would love to but it's going to take a while since I'm still very new to Rust and this project. 😬 

> Otherwise, if you can grant me permission to push to this branch we can work on these together.

Feel free to push to this branch whenever. Maybe you could implement `chr` and `allbits?`, while I implement the other two, or whichever you prefer as long as I could learn something along.

**Comment by lopopolo** [comment]
Sounds good. GH-186 is already open for `Integer#chr`. I’ll try to implement `allbits?` first since the others should be similar and you can use it as a guide.

I’ll ping back here once I’ve got it ready.

Thanks again for all your work on this 💎

**Comment by lopopolo** [comment]
GH-186, implement `Integer#chr`, was implemented in GH-192. ✅ 

A rebase should pick it up.

**Comment by lopopolo** [comment]
Thanks for rebasing @tkbky. I'm merging this since the specs still pass.

Filed GH-195 for following up with `Integer#allbits?`, `Integer#anybits?`, and `Integer#nobits?`.

@tkbky it looks like these functions are able to be implemented in pure Ruby. Do you want to pickup the ticket?

**Comment by tkbky** [comment]
@lopopolo Thanks for merging this in.

> it looks like these functions are able to be implemented in pure Ruby. Do you want to pickup the ticket?

Yes, I would love to.

**Pull request merged**